src/analysis/ref_mode.rs

```diff
@@ -27,7 +27,11 @@ impl RefMode {
             ..
         }) = env.config.objects.get(&tid.full_name(&env.library))
         {
-            return ref_mode;
+            if direction == library::ParameterDirection::In {
+                return ref_mode;
+            } else {
+                return RefMode::None;
```

Comment:
Caps in gstreamer. Manual but ref-counted, so `ref_mode = "ref"` is configured. And if you look below in that function, the behaviour is the same. It only returns something else than `None` for `In` parameters, as everything else makes no sense.

Without this, e.g. a property returning Caps will return `Option<&Caps>`, which makes no sense and is nothing that could be constructed.